dist/api_data.py

```diff
@@ -682,6 +682,7 @@ def __ge__(self, other):
             'api',
             'block',
             'checkpoint',
+            'checkpoint_garbage_collection',
```

Comment:
I don't feel strongly, but given that the tests are all `test_gc##` and such, I think `checkpoint_gc` is equally as clear and shorter.